pkg/api/api.go

```diff
@@ -166,6 +166,12 @@ func Router() *gin.Engine {
 			{
 				dashboard.GET("", GetDashboardData)
 			}
+			oidcClient := authorizedv1.Group("/oidcClients")
+			{
+				oidcClient.GET("", middleware.RoleBasedAccessMiddleware([]string{"ADMIN"}), GetUserOidcClients)
+				oidcClient.POST("", middleware.RoleBasedAccessMiddleware([]string{"ADMIN"}), AddOidcClient)
+				oidcClient.DELETE("", middleware.RoleBasedAccessMiddleware([]string{"ADMIN"}), RevokeClient)
```

Comment:
These endpoints should be allowed to called by all users so they can add their client-ids.